packages/grafana-prometheus/src/language_provider.ts

```diff
@@ -416,7 +420,7 @@ export default class PromQlLanguageProvider extends LanguageProvider implements
   fetchSeriesLabelsMatch = async (
     timeRange: TimeRange,
     name: string,
-    withLimit?: string
+    withLimit?: string | number
```

Comment:
i'm curious why some of these are type: `string | number` and others are just `number`

Reply:
Number must be the one but for backward compatibility I just did that. We will remove all those methods anyway. 
For the one that I didn't add, there we use `URLSearchParams` and it just needs string. I don't want to change the logic just because of a short lived refactor.